includes/sys_menu.php

```diff
@@ -55,10 +55,18 @@ function make_navigation()
 {
     $page = current_page();
     $menu = [];
+
+    # figure out if the menu is displayed to an admin account or a regular user
+    # and pick the correct name for the shift overview link
+    $shift_menu_name = 'general.my_shifts';
```

Comment:
the implementation is fine, sadly it makes no sense for this page (see other review comments)